backend/src/v2/cmd/driver/main.go

```diff
@@ -100,6 +110,15 @@ func validate() error {
 	if *driverType == "" {
 		return fmt.Errorf("argument --%s must be specified", driverTypeArg)
 	}
+	if *httpProxy == unsetProxyArgValue {
+		return fmt.Errorf("argument --%s is required but can be an empty value", httpProxyArg)
+	}
+	if *httpsProxy == unsetProxyArgValue {
+		return fmt.Errorf("argument --%s is required but can be an empty value", httpsProxyArg)
+	}
+	if *noProxy == unsetProxyArgValue {
+		return fmt.Errorf("argument --%s is required but can be an empty value", noProxyArg)
+	}
```

Comment:
why are these required?

Reply:
I made them required to avoid not setting them by mistake.

Reply:
Also, if they were optional, we would have to run the default logic at these points again.